lib/ui/src/components/sidebar/Heading.tsx

```diff
@@ -34,13 +36,36 @@ const HeadingWrapper = styled.div({
   position: 'relative',
 });
 
+const SkipToCanvasLink = styled(Button)(({ theme }) => ({
+  display: 'none',
+  '@media (min-width: 600px)': {
+    display: 'block',
+    position: 'absolute',
+    width: '100%',
+    padding: '10px 15px',
+    fontSize: theme.typography.size.s1,
+    zIndex: 1,
+    transform: 'translate(0,-100px)',
+    '&:focus': {
+      transform: 'translate(0,-10px)',
```

Comment:
I might change the focused positioning to `translate(0, 0)`  or  `translateY(0)` so it aligns  over the logo:

<img width="234" alt="image" src="https://user-images.githubusercontent.com/1123119/130139670-0a227497-08ee-423a-bdbc-17c84ab01190.png">